test/lib/ufe/testParentCmd.py

```diff
@@ -470,3 +473,14 @@ def checkUnparent(done):
 
             cmds.redo()
             checkUnparent(done=True)
+
+    def testParentingToGPrim(self):
+        '''Parenting an object to UsdGeomGprim object is not allowed'''
+
+        # open tree scene
+        mayaUtils.openTreeScene()
+
+        with self.assertRaises(RuntimeError):
+            cmds.parent("|Tree_usd|Tree_usdShape,/TreeBase/trunk",
+                        "|Tree_usd|Tree_usdShape,/TreeBase/leavesXform/leaves")
+        
```

Comment:
Added a unit test to back the Gprim parenting restriction.

Reply:
@ppt-adsk  To make testParentRelative, testParentAbsolute working under the new rule, I created a top level Xforms for cube, cylinder, sphere in "simpleSceneTRS.usda" and made sure the Translate/Rotate/Scale and Order Ops matches the underlying meshes. Github won't show the diff results. You need to use your diff-editor locally to see the changes.